docs/calamari/DevelopOnDolphin.md

```diff
@@ -1,96 +1,91 @@
 # 🛠 Develop on Dolphin
 
-## Quick Overview
-This guide outlines the steps needed to create a Calamari node for testing your applications on our public testnet name Dolphin.
-
-A Calamari development node is your own personal development environment for building and testing applications on Calamari Network. It enables you to get started quickly and easily without the overhead of a relay chain. By default a block will be created when a transaction is received.
-
-To get started running a Calamari node you will need to first download it or build it. This could take roughly 30 minutes or longer to complete depending on your hardware.
+## Introduction
+This guide provides step-by-step instructions to set up a Calamari development node to test your applications on the public testnet named Dolphin. A Calamari development node acts as your personal development environment to build and test applications on the Calamari Network. It eliminates the need for a relay chain and creates blocks automatically when transactions are received.
 
 ## Download Binary File
 
-To download any of the official releases of Calamari, please go to our [release page](https://github.com/Manta-Network/Manta/releases).
+To download an official release of Calamari, visit our [release page](https://github.com/Manta-Network/Manta/releases).
 
 ## Build Binary File
 
-First, start by cloning a specific tag of the Calamari repo that you can find here:
-
-`https://github.com/Manta-Network/Manta/`
+1. Clone the specific tag of the Calamari repository from this URL: `https://github.com/Manta-Network/Manta/`
 
-```
+```sh
 git clone -b v4.0.1 https://github.com/Manta-Network/Manta
 cd Manta
 ```
 
-If you already have Rust installed, you can skip the next two steps. Otherwise, install Rust and its prerequisites via Rust's recommended method by executing:
-`curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh`
+2. Clone the specific tag of the Calamari repository from this URL:
+```sh
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
 
-Next, update your PATH environment variable by running:
-`source $HOME/.cargo/env`
+3. Update your PATH environment variable with this command:
+```sh
+source $HOME/.cargo/env
+```
 
-Now you can either build and run the binary separately with:
-```bash
+4. To build and run the binary file, choose one of the following options:
+```sh
   cargo build --release
   ./target/release/manta --chain=calamari-localdev --alice --tmp
```

Comment:
The name of chain spec is not correct.
`calamari-localdev`.

Reply:
what do you mean ? That's the chain-spec for the dev node.

Reply:
I didn't notice we have this chain spec, oops